parent/pom.xml

```diff
@@ -620,7 +620,7 @@
     <repository>
       <id>com.springsource.repository.bundles.external</id>
       <name>SpringSource Enterprise Bundle Repository - External Bundle Releases</name>
-      <url>http://repository.springsource.com/maven/bundles/external</url>
+      <url>https://repository.springsource.com/maven/bundles/external</url>
```

Comment:
@rwinch Do you guys own this too? Why does this still resolve?

Reply:
For others seeing this comment. We took this offline, but we are looking into other URLs we have and disabling http access See https://spring.io/blog/2019/09/16/goodbye-http-repo-spring-use-https